main.go

```diff
@@ -35,10 +35,18 @@ import (
 )
 
 var (
+	// scheme holds a new scheme for the operator
 	scheme   = runtime.NewScheme()
+
+	// setupLog will be used for logging the operator "setup" process so that users know
+	// what parts of the logging are associated with the setup of the manager and
+	// controller
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+// init sets up the Go client and NFD schemes. The function "utilruntime.Must" is used to
+// panic on non-nil errors that could occur when adding the scheme, as opposed to just letting
+// an error occur without properly handling it.
```

Comment:
Suggestion: this comment could be moved one liner lower (and edit a bit to match the context and be more concise)

E.g. something like 
```
Set up the Go client and NFD schemes. Panic on errors.
```

Reply:
Changed the comment to now read "Set up the Go client and NFD schemes. Panic on errors."